src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -145,7 +145,7 @@ public SortedSet(IEnumerable<T> collection, IComparer<T> comparer)
                     int index = 1;
                     for (int i = 1; i < count; i++)
                     {
-                        if (comparer.Compare(els[i], els[i - 1]) != 0)
+                        if (_comparer.Compare(els[i], els[i - 1]) != 0)
```

Comment:
I'd assume that the reason `comparer` is used is to avoid the repeated field access. Maybe assign `comparer = _comparer` (with a comment because it would look a bit strange) and keep is line as is?